src/frontend/next/src/pages/_app.tsx

```diff
@@ -1,6 +1,7 @@
 import { useEffect } from 'react';
 import { AppProps } from 'next/app';
 import '../styles/globals.css';
+import 'highlight.js/styles/github.css';
```

Comment:
cc @PedroFonsecaDEV, this is the file we need to have in our global CSS for syntax highlighting to work.

We can remove this from here.

Reply:
Okay. Working on it.

Reply:
@humphd done: 
Pull: https://github.com/Seneca-CDOT/telescope/pull/1579